cmd/laas/main.go

```diff
@@ -68,9 +72,14 @@ func main() {
 
 	r := gin.Default()
 	r.NoRoute(api.HandleInvalidUrl)
-	r.GET("/api/licenses", api.GetAllLicense)
-	r.GET("/api/license/:shortname", api.GetLicense)
-	r.POST("/api/license", api.CreateLicense)
-	r.PATCH("/api/license/:shortname", api.UpdateLicense)
+	authorized := r.Group("/")
+	authorized.Use(auth.AuthenticationMiddleware())
+	authorized.GET("/api/license/:shortname", api.GetLicense)
+	authorized.POST("/api/license", api.CreateLicense)
+	authorized.PATCH("/api/license/update/:shortname", api.UpdateLicense)
```

Comment:
Please use consistent plural form for the endpoints. It should be `/api/licenses/update/:shortname` or `/api/licenses/:shortname` not `/api/license/:shortname`.

You can work on it in a separate PR if needed.